docs/src/topics/flavors/cluster-autoscaler.md

```diff
@@ -0,0 +1,40 @@
+# Cluster Autoscaler
+
+## Specification
+
+| Control Plane | CNI    | Default OS   | Installs ClusterClass | Autoscaling |
```

Comment:
I think for the columns we should stick to the columns used in the default flavor and extra capabilities should be mentioned above this Specification section (e.g. https://linode.github.io/cluster-api-provider-linode/topics/flavors/etcd-disk.html)